canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.scss

```diff
@@ -20,8 +20,21 @@
 	}
 	.bx--checkbox-wrapper { //override carbon checkbox styling TODO currently doesn't override
 		margin-bottom: $spacing-03;
+		margin-right: $spacing-03;
```

Comment:
This should be removed since the tooltip already has the spacing.

Reply:
removed it.